src/App.js

```diff
@@ -23,9 +24,8 @@ function App() {
           <Route path="/" element={<Home />} />
           <Route path="/categories" element={<Categories />} />
         </Routes>
-        <h2>Bookstore</h2>
-        <BookForm />
         <BookList />
+        <BookForm />
       </div>
     </Router>
```

Comment:
- [ ] Awesome work on your project! I did notice a small problem, though. When I navigate to the categories page, it shows both the "Under Construction" message and the book listings from the books tab. I've captured it in the video below for you to take a look.

   [scrnli_2_9_2023_23-32-34.webm](https://github.com/jlberbesi/react-bookstore/assets/107952724/0ed7ef16-a38c-490d-a6e7-e906fce627cc)

Reply:
@taldr27 thanks for your feedback. Guess already fixed it.